src/remixAppManager.js

```diff
@@ -202,6 +202,18 @@ export class RemixAppManager extends PluginEngine {
       icon: 'https://zokrates.blockchain-it.hr/zokrates.svg',
       location: 'sidePanel'
     }
+    const ink = {
```

Comment:
Could you add the plugin reference directly in https://github.com/ethereum/remix-plugins-directory ?
cc @GrandSchtroumpf

Reply:
Sure.

In here: https://github.com/ethereum/remix-plugins-directory/tree/master/plugins
and here: https://github.com/ethereum/remix-plugins-directory/blob/master/build/profile.json

Anywhere else?

Reply:
just add a new folder in there https://github.com/ethereum/remix-plugins-directory/tree/master/plugins
with the profile.json

Reply:
Will do, tnx!